index.html

```diff
@@ -0,0 +1,251 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Abdel-Khafid's Portfolio</title>
+    <style>
+        @import url('https://fonts.cdnfonts.com/css/poppins');
+</style>
+<link href='https://fonts.googleapis.com/css?family=Playfair Display' rel='stylesheet'>
+<link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+   <header>
+    <nav class="navbar">
+        <div class="hamburger">
+            <img src="./assets/ic_menu.svg" alt="hamburger">
+        </div>
+        <div class="menu">
+            <ul class="sections">
+                <li><a href="#intro">Hello</a></li>
+                <li><a href="#projects">Projects</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+
+        </div>
+
+    </nav>
+   </header> 
+   <section id="intro">
+    <div class="intro">
+        <h1>Abdel-Khafid SALAOU</h1>
+        <p>
+            Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.
+        </p>
+        <p>
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <button class="collab-btn">Start Collaboration</button>
+        <a href=""><img class="img-intro-bottom" src="./assets/mobile/intro_bgrd_bottom.svg" alt="graphic elements for design"></a>
```

Comment:
According to the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), you should avoid lines of code that are too long (100 characters or more). [check here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Avoid%20lines%20of%20code%20that%20are%20too%20long%20(100%20characters%20or%20more).)